jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +441,59 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(BrowserEvents.CLICK){
+
+            @Override
+            public void onBrowserEvent(Context context, Element elem, ProcessInstanceSummary val, NativeEvent event,
+                    ValueUpdater<ProcessInstanceSummary> valueUpdater) {
+                super.onBrowserEvent(context, elem, val, event, valueUpdater);
+                if(BrowserEvents.CLICK.equals(event.getType()) && val != null) {
+                    errCountPopup.setData(val.getProcessInstanceId(),  val.getErrorCount());
+                    errCountPopup.showAt(event.getClientX(), event.getClientY());
```

Comment:
Since showAt and setData are only used in here, I would suggest moving it to a single method with all necessary parameters.